File: android/app/src/main/java/net/ipv64/kivop/dtos/PosterServiceDTOs/TakeDownPosterPositionResponseDTO.kt
package net.ipv64.kivop.dtos.PosterServiceDTOs

import java.util.UUID
import java.time.LocalDateTime

data class TakeDownPosterPositionResponseDTO (
    var posterPosition : UUID,
    var removedAt : LocalDateTime,
    var removedBy : UUID,
    var image : String,
)

File: DTOs/Sources/PosterServiceDTOs/TakeDownPosterPositionResponseDTO.swift
import Foundation

public struct TakeDownPosterPositionResponseDTO: Codable {
    public var posterPosition: UUID
    public var removedAt: Date
    public var removedBy: UUID
    
    public init(posterPosition: UUID, removedAt: Date, removedBy: UUID) {
        self.posterPosition = posterPosition
        self.removedAt = removedAt
        self.removedBy = removedBy
    }
}
